compiler/rustc_session/src/config.rs

```diff
@@ -86,12 +86,18 @@ pub enum CFProtection {
 
 #[derive(Clone, Copy, Debug, PartialEq, Hash, HashStable_Generic)]
 pub enum OptLevel {
-    No,         // -O0
-    Less,       // -O1
-    Default,    // -O2
-    Aggressive, // -O3
-    Size,       // -Os
-    SizeMin,    // -Oz
+    /// `-O0`
```

Comment:
```suggestion
    /// `-Copt-level=0`
```
these flags do not exist, let's name them correctly